homeassistant/components/overkiz/button.py

```diff
@@ -85,6 +104,13 @@ async def async_setup_entry(
 class OverkizButton(OverkizDescriptiveEntity, ButtonEntity):
     """Representation of an Overkiz Button."""
 
+    entity_description: OverkizButtonDescription
+
     async def async_press(self) -> None:
         """Handle the button press."""
-        await self.executor.async_execute_command(self.entity_description.key)
+        if self.entity_description.press_args:
+            await self.executor.async_execute_command(
+                self.entity_description.key, self.entity_description.press_args
+            )
+        else:
+            await self.executor.async_execute_command(self.entity_description.key)
```

Comment:
Not that it makes much difference but you could do

```suggestion
            return
        await self.executor.async_execute_command(self.entity_description.key)
```

Reply:
Which code style would be preferred in core? I don't mind :)

Reply:
The one I suggested 😉

Reply:
Fixed in 19bfe26ed3b8620d214534d39dd8c58efca05bd6